main.css

```diff
@@ -145,8 +166,14 @@ a {
 .p-8 {
   padding: 8px;
 }
+.pd-l-10 {
+  padding-left: 10px;
+}
+.bg-color {
+  background-color: #f7941e;
```

Comment:
Research about CSS custom properties (aka CSS variables) and move this color to a variable.